sw/device/tests/sensor_ctrl_alerts.c

```diff
@@ -47,7 +48,7 @@ static dif_alert_handler_t alert_handler;
  * so we go from 0xFFFF_FFFF -> 0xFFFF_FFFE -> 0xFFFF_FFFC
  */
 __attribute__((section(".non_volatile_scratch")))
-const volatile uint32_t events_vector = -1;
+const volatile uint32_t events_vector = UINT32_MAX;
```

Comment:
nit: Can this be non-`const`?

Reply:
hmmm....actually why is const there.. that doesn't seem right.